### DISCUSSION TITLE
Suggestion for Spanish Translation

### DISCUSSION THREAD

**Discussion by Naradiel** [discord-tickets/bot#443] [Support (Q&A)] [answered]
Hi there!

I hope this message finds you well. I'm Naradiel, a junior programmer who has been actively using your Discord ticket bot. I have been impressed by its functionality and user-friendliness. However, I noticed that it is currently only available in English. I would like to offer my assistance in translating the bot into Spanish to enable a broader audience to benefit from it.

Thank you for your attention!

**Comment by eartharoid** [comment, marked as answer]
I have added Spanish at https://hosted.weblate.org/projects/discord-tickets/bot/es/ but you can only add suggestions.
Please login with GitHub and DM me (on Discord) your email address so I can add you as a reviewer.
